ui/src/views/Shell/index.jsx

```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
 import { withStyles } from '@material-ui/core/styles';
 import ShellConsole from '../../components/Shell';
-import Dashboard from '../../components/Dashboard';
```

Comment:
What's nice about using the Dashboard component is that it sets, the logo icon and page title in the browser tab:

![Screen Shot 2019-11-18 at 9 05 53 AM](https://user-images.githubusercontent.com/3766511/69058990-aa248380-09e2-11ea-854b-2992fa5428ba.png)

We should probablty still use the Dashboard component but expose a new prop `disableAppbar` that hides the app bar if the prop is provided:

```diff
diff --git a/ui/src/components/Dashboard/index.jsx b/ui/src/components/Dashboard/index.jsx
index b438f9a10..fb96f0b75 100644
--- a/ui/src/components/Dashboard/index.jsx
+++ b/ui/src/components/Dashboard/index.jsx
@@ -165,6 +165,7 @@ export default class Dashboard extends Component {
     helpView: null,
     docs: false,
     disableTitleFormatting: false,
+    disableAppbar: false,
   };
 
   static propTypes = {
@@ -199,6 +200,10 @@ export default class Dashboard extends Component {
      * If true, the title will not be formatted to uppercase.
      */
     disableTitleFormatting: bool,
+    /**
+     * If true, the app bar will not be rendered.
+     */
+    disableAppbar: bool,
   };
 
   static getDerivedStateFromError(error) {
```

Reply:
@helfi92 I made the change and introduced the new prop, How do I make the use of the prop to hide the menu ?

Reply:
If `disableAppbar` is `true`, then do not render the app bar (i.e., `<Appbar .. />`). Else, render the app bar.